### PR TITLE
Fix most horrible typos

### PR DESCRIPTION
**Pull request by JuliaSprenger**
There were not enough accents in the text. Now it is good french!

### CODE REVIEW

**Comment by tobikausk** [comment]
I like!

File: quotes.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Ils sont fous, ces Romains ! - by ???
+Ils sont fousé, cés româins ! - by Asterix
```
Comment (JuliaSprenger) [inline quotes.txt]: I don't like this looks ugly. But thanks for the contribution!

Comment (bjoern1001001) [inline quotes.txt]: The first version was better I think.

**Comment by JuliaSprenger** [comment]
Thanks! #1 did something similar.

File: quotes.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-Ils sont fous, ces Romains ! - by ???
```
Comment (bjoern1001001) [inline quotes.txt]: Test

Comment (tobikausk) [inline quotes.txt]: Friede den Huetten!

File: quotes.txt (hunk 1 above)
Comment (tobikausk) [inline quotes.txt]: Krieg den Palaesten!

**Comment by GinSch** [comment]
Hi

**Comment by EmanueleLucrezia** [comment]
Jsp4president!

**Pull request merged**